src/Illuminate/Broadcasting/Broadcasters/AblyBroadcaster.php

```diff
@@ -204,32 +310,34 @@ protected function formatChannels(array $channels)
     }
 
     /**
-     * Get the public token value from the Ably key.
+     * Build an Ably message object for broadcasting.
      *
-     * @return mixed
+     * @param $event string
+     * @param $payload array
```

Comment:
```suggestion
     * @param  string  $event
     * @param  array  $payload
```
First variable type and then the variable name

Reply:
Fixed!